### ISSUE TITLE
Simplify OAuth callback URL

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis-ui#870]
We use `https://$SYNDESIS/api/v1/credentials/callback` as the URL for OAuth callbacks, perhaps we could route in nginx by existence of a cookie to that URL making the callback URL simply `https://$SYNDESIS`.

**Comment by rhuss** [comment]
That would be awesome as this is a datum a user has to manually copy over.

**Comment by rhuss** [comment]
@jimmidyson guru of nginx and everything, do you think nginx can do this ?

**Comment by zregvart** [comment]
@rhuss totally: https://serverfault.com/questions/268633/controlling-nginx-proxy-target-using-a-cookie

**Comment by jludvice** [comment]
Do you think the exact URL really maters? Isn't just string that will be copy-pasted from settings page to twitter/salesfoce setup page.

Having url like `/api/v1/credentials/callback` makes it quite clear it's callback url.
Misbehaving auth might be quite challenging to debug if we loose the "routing cookie" and requests will be routed incorrectly.

**Comment by jludvice** [comment]
CC @dsimansk

**Comment by zregvart** [comment]
Perhaps we could monitor the requests and see that OAuth callbacks are ending at `/` and not being redirected to UI return URL. Just a thought.

**Comment by gashcrumb** [comment]
Yeah, to be honest it'd be better to just display the callback URL that would be required with a little clippy dealio on the settings page and not worry about the actual URL so much.
